front/src/common/validators/form.validator.js

```diff
@@ -28,21 +28,21 @@ export function validatePhoneNumber(_, value) {
   }
 }
 export function validateEmail(_, value) {
-    const emailRegex = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/;
+  const emailRegex = /^\w+([.-]?\w+)*@\w+([.-]?\w+)*(\.\w{2,3})+$/;
```

Comment:
Don`t understand the reason of this editing

Reply:
with this backslash I had warnings with React